docs/how_to_build_images.md

```diff
@@ -0,0 +1,36 @@
+# How to build images
+
+Use appropriate playbook inside `playbooks` directory to build the images you need.
+
+To build localhost/foreman:2.1-42 image:
+
+```
+ansible-playbook playbooks/build-foreman-image.yml -e version=2.1 -e version_build=42
+```
+
+To build awesome-repo/foreman:2.1-1 and push into the registry:
+
+```
+ansible-playbook playbooks/build-foreman-image.yml -e version=2.1 -e registry=awesome-repo -e registry_user=admin -e registry_token=changeme -e push=true
+```
+
+It is possible to use additional snippets to modify an image build which is useful when you need to copy additional files into an image:
+
+```
+touch roles/foreman-image/files/container-assets/foo.yml
```

Comment:
Note here you touch a file in `files`, and down below this you cat it into `templates`.